src/components/Table.styled.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
+
+export const StyledDiv = styled.div`
+  overflow-x: auto;
+  padding-bottom: 15px;
```

Comment:
```suggestion
  padding-bottom: ${spacings.MEDIUM};
```
You could import `spacings` and use what we have defined as medium, aka 16px. I've defined these based on [EDS Guildelines](https://eds.equinor.com/0b0c666ab/p/721e1f-spacing/b/0899da). Then, if EDS decides to change their definitions of large, medium etc, we won't have to find and change all size definitions around our code.